termwiz/src/escape/apc.rs

```diff
@@ -301,6 +301,15 @@ fn read_shared_memory_data(
     Ok(data)
 }
 
+#[cfg(all(unix, target_os = "android"))]
+fn read_shared_memory_data(
+    _name: &str,
+    _data_offset: Option<u32>,
+    _data_size: Option<u32>,
+) -> std::result::Result<std::vec::Vec<u8>, std::io::Error> {
+    panic!("Not supported on Android: read_shared_memory_data");
```

Comment:
Please make this return a std::io::Error with https://doc.rust-lang.org/std/io/enum.ErrorKind.html#variant.Unsupported

Reply:
Ah I didn't know this variant exist. Lemme change now. Thanks!

Reply:
Changed to use that error variant. Marking as resolved.